data-pipeline/src/data_pipeline/data_types/gene.py

```diff
@@ -226,6 +226,15 @@ def import_hgnc(path):
     return ds
 
 
+def prepare_table_for_release(genes_path, keep_mane_version_global_annotation):
+    ds = hl.import_table(genes_path)
+    if keep_mane_version_global_annotation:
+        globals_dict = ds.index_globals()
+        ds = ds.select_globals(mane_select_version=globals_dict["annotations"]["mane_select_transcript"]["version"])
+    ds = ds.select_globals()
```

Comment:
this `select_globals` statement needs to be in an `else` or the globals kept from line 233 will be removed, right?